pkg/sbom/spdx/unmarshal.go

```diff
@@ -246,3 +259,30 @@ func parseSourceInfo(pkgType, sourceInfo string) (epoch int, name, ver, rel stri
 	}
 	return epoch, name, ver, rel, nil
 }
+
+// getPackageFilePaths parses Relationships and finds filepaths for packages
+func getPackageFilePaths(spdxDocument *spdx.Document) map[string]string {
+	packageFilePaths := map[string]string{}
+	fileSPDXIdentifierMap := createFileSPDXIdentifierMap(spdxDocument.Files)
+	for _, rel := range spdxDocument.Relationships {
+		if rel.Relationship == common.TypeRelationshipDescribe || rel.Relationship == "DESCRIBE" {
```

Comment:
We can care about `CONTAINS` only, right?


```suggestion
		if rel.Relationship != common.TypeRelationshipContains && rel.Relationship != "CONTAIN" {
```

Reply:
Right! Made in https://github.com/aquasecurity/trivy/pull/4513/commits/01bb6e38d3d8f6d560300e02404b218ee2bd7111